docs/advanced/lanes.md

```diff
@@ -305,4 +305,4 @@ for_platform :mac do
 end
 ```
 
-you can expect the `app_identifier` to equal `"com.forplatform.mac.forlane.release"` when invoking `fastlane mac release`.
\ No newline at end of file
+you can expect the `app_identifier` to equal `"com.forplatform.mac.forlane.release"` when invoking `fastlane mac release`.
```

Comment:
What changed on this line? 🤔

Reply:
Probably added a newline that wasn't there after I touched the file on Windows?